index.rst

```diff
@@ -1,51 +1,387 @@
+I use :eq:`eqn-basic_template` to build a DCR-matched template for the simulated observation (:numref:`fig-sim_template`), and subtract this template to make the difference image (:numref:`fig-sim_template_diff`).
+For comparison, in :numref:`fig-sim_image_diff` I subtract a second simulated image generated with the same field 10 degrees closer to zenith.
+
+.. figure:: /_static/simulations/simulated_108_image.png
+   :name: fig-sim_image
+
+   Simulated g-band image with airmass 1.3
+
+.. figure:: /_static/simulations/simulated_image_108_template.png
+   :name: fig-sim_template
+
+   The DCR-matched template for the simulated image in :numref:`fig-sim_image`.
+
+.. figure:: /_static/simulations/simulated_image_108_template_difference.png
+   :name: fig-sim_template_diff
+
+   Difference image of :numref:`fig-sim_image` - :numref:`fig-sim_template`.
+
+.. figure:: /_static/simulations/simulated_image_108_110_difference.png
+   :name: fig-sim_image_diff
+
+   Difference image of :numref:`fig-sim_image` with another simulation of the same field 10 degrees closer to zenith (airmass 1.22).
+
+Examples with DECam images
+--------------------------
+
+For a more rigorous test, I have also built DCR-matched templates for `DECam HiTS <https://arxiv.org/abs/1609.03567>`_ observations, which were calibrated and provided by Francisco Förster. 
+For these images I used the implementation outlined above using the simplified equation :eq:`eqn-iterative_sum`, despite the images having variable seeing.
+Because I have not yet implemented :eq:`eqn-psf_iterative_sum` using measured PSFs for each image, I have simply excluded observations with PSF FWHWs greater than 4 pixels (2.5 - 4 pixel widths are common). 
+However, it should be noted that the images with PSFs at the larger end of that range are not as well matched by their templates, so restricting the input images going into the model to those with good seeing may not be sufficient in the future.
+
+.. figure:: /_static/Decam/0410998_image.png
+   :name: fig-decam_image
+
+   Decam observation 410998, 'g'-band with airmass 1.33.
+
+.. figure:: /_static/Decam/0410998_template.png
+   :name: fig-decam_template
+
+   The DCR-matched template for 410998, constructed from 12 observations with airmass ranging between 1.13 and 1.77.
+   Note that the noise level is significantly decreased.
+
+.. figure:: /_static/Decam/0410998-template_difference.png
+   :name: fig-decam_template_diff
+
+   Difference image of :numref:`fig-decam_image` - :numref:`fig-decam_template`.
+
+.. figure:: /_static/Decam/0410998-0411232_difference.png
+   :name: fig-decam_image_diff
+
+   Difference image of :numref:`fig-decam_image` with a second DECam observation taken approximately 10 degrees closer to zenith (at airmass 1.23).
+
+The DCR Sky Model
+=================
+
+
+Simulated source spectra
```

Comment:
This section would be great to have.  I think it will really nail it down for people if you can show you are getting the right answer out.